### PR TITLE
chore: alpha Menu fix merge

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [x] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

Style patch of new Menu.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by shaodahong — APPROVED** [review]

**Review by xiaohuoni — APPROVED** [review]

File: components/date-picker/index.en-US.md [hunk 1]
```diff
@@ -77,10 +77,10 @@ The following APIs are shared by DatePicker, RangePicker.
...
 ### Common Methods
 
-| Name | Description | Version |
-| --- | --- | --- |
-| blur() | Remove focus |  |
-| focus() | Get focus |  |
+| Name    | Description  | Version |
+| ------- | ------------ | ------- |
+| blur()  | Remove focus |         |
+| focus() | Get focus    |         |
```
Comment (hengkx) [inline components/date-picker/index.en-US.md:84]: 这个怎么 也改了

Comment (zombieJ) [reply components/date-picker/index.en-US.md:84]: 没动过……是不是 prettier hooks 做的？

Comment (hengkx) [reply components/date-picker/index.en-US.md:84]: 有可能 表格的 格式化 用的是 `sort-api`

Comment (xrkffgg) [reply components/date-picker/index.en-US.md:84]: 可能是 vscode 插件

**Pull request merged**